index.html

```diff
@@ -5987,7 +5987,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 &lt;span&gt; Sample Content &lt;a href="...">let's go!&lt;/a> &lt;/span&gt;
 </pre>
 				<p>In HTML, the <code>&lt;img&gt;</code> <a>element</a> is treated as a single entity regardless of the type of image file. Consequently, using <code>role="none"</code> or <code>role="presentation"</code> on an HTML <code>img</code> is equivalent to using <code>aria-hidden="true"</code>. In order to make the image contents accessible, authors can embed the object using an <code>&lt;object&gt;</code> or <code>&lt;iframe&gt;</code> <a>element</a>, or use inline <abbr title="Scalable Vector Graphics">SVG</abbr> code, and follow the accessibility guidelines for the image content.</p>
-				<p>Authors SHOULD NOT provide a meaningful text alternative (for example, use <code>alt=""</code> in HTML) when the <code>none</code>/<code>presentation</code> role is applied to an image.</p>
+				<p>Authors SHOULD NOT provide a text alternative when the <code>none</code>/<code>presentation</code> role is applied to an image.</p>
```

Comment:
I'm not sure that this was trying to say that. 
I think it was saying don't use alt="something"
But it is fine (although redundant) to use alt="" with role="none" as they essentially do the same thing. Indeed the example below does exactly that.

```suggestion
				<p>Authors SHOULD NOT provide a non empty text alternative when the <code>none</code>/<code>presentation</code> role is applied to an image.</p>
```

Reply:
I take the point regarding the following example but since this will get flagged by nu validator already, we should perhaps remove it.

I find the double negation unnecessarily complicated. Since empty aria-label cause validation errors, I think it wouldn't reduce confusion as much so I think it's better as is. But it's not really that important to me - happy to merge the suggestion.

Reply:
FWIW I don't think `alt="" role="none"` should fail the validator. I think it should be okay to do this. 

It should also be fine to do `role="none"` or `alt=""`